net/haproxy/src/opnsense/mvc/app/controllers/OPNsense/HAProxy/forms/dialogBackend.xml

```diff
@@ -131,6 +131,20 @@
         <advanced>true</advanced>
     </field>
     <field>
+        <id>backend.tuning_defaultserver</id>
+        <label>Default for server</label>
+        <type>text</type>
+        <help><![CDATA[Default option for all server entries.]]></help>
+        <advanced>true</advanced>
+    </field>
+    <field>
+        <id>backend.tuning_noport</id>
+        <label>Don't use port on server</label>
```

Comment:
I'd prefer a shorter "label". What about "Use Frontend port" instead?
The help text does provide a good explanation, so we should keep labels short.